Source/Core/Gen3/Searchers/StaticSearcher3.hpp

```diff
@@ -63,10 +64,11 @@ class StaticSearcher3 : public StaticSearcher<Profile3, StateFilter, SearcherSta
      * @param spd SpD IV
      * @param spe Spe IV
      * @param staticTemplate Pokemon template
+     * @param buggedRoamer FRLG/RS roamer
      *
      * @return Vector of computed states
      */
-    std::vector<SearcherState> search(u8 hp, u8 atk, u8 def, u8 spa, u8 spd, u8 spe, const StaticTemplate *staticTemplate) const;
+    std::vector<SearcherState> search(u8 hp, u8 atk, u8 def, u8 spa, u8 spd, u8 spe, const StaticTemplate *staticTemplate, const bool buggedRoamer) const;
```

Comment:
Won't need to be passed in with the change I suggested

Reply:
Changed